mobile/app-identifier.ts

```diff
@@ -193,6 +193,32 @@ export class IOSNativeScriptCompanionAppIdentifier implements Mobile.IAppIdentif
 
 }
 
+export class WP8CompanionAppIdentifier implements Mobile.IAppIdentifier {
+	get appIdentifier(): string {
+		return "";
```

Comment:
Why this is empty string? It is strange to me wp8 companion app identifier to be empty string...